### PR TITLE
xen: Add patches to xen for dom0less

### PR DESCRIPTION
**Pull request by dsemenets**
Add patch series to xen. This series is trying to fix the reported guest magic region allocation issue for 1:1 direct-mapped Dom0less domUs.

### CODE REVIEW

File: recipes-extended/xen/xen-source.inc [hunk 1]
```diff
@@ -1,5 +1,13 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
 LIC_FILES_CHKSUM = "file://COPYING;md5=d1a1e216f80b6d8da95fec897d0dbec9"
 
 XEN_URL = "git://github.com/xen-troops/xen.git;protocol=https;branch=xen-4.19-xt0.1"
 XEN_REL = "4.19"
 XEN_REV = "${AUTOREV}"
+
```
Comment (lorc) [inline recipes-extended/xen/xen-source.inc]: This does not belong to `xen-source.inc`

Please add `xen.inc` and move this append there.

File: recipes-extended/xen/xen_git.bbappend [hunk 2]
```diff
@@ -1,5 +1,3 @@
-FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
-
```
Comment (lorc) [inline recipes-extended/xen/xen_git.bbappend:2]: Don't do this

**Review by lorc — CHANGES_REQUESTED** [review]
Please provide comprehensive commit message, not copy-paste from added .patch files. 

Also, you commit header is too generic.

**Comment by GrygiriiS** [comment]
I checked this and patch is doing what is required in general. I leave decision about "How should be done in the right way" to @lorc

**Comment by lorc** [comment]
Well, I clearly wrote 

> Please add `xen.inc` and move this append there.

Please contact me if you need further clarification

**Review by lorc — APPROVED** [review]
Thanks

`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by oleksiimoisieiev** [comment]
LGTM `Acked-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>`

File: recipes-extended/xen/xen-source.inc (hunk 1 above)
Comment (dsemenets) [reply recipes-extended/xen/xen-source.inc]: Done

File: recipes-extended/xen/xen_git.bbappend (hunk 2 above)
Comment (dsemenets) [reply recipes-extended/xen/xen_git.bbappend:2]: Ok

**Pull request merged**